### DISCUSSION TITLE
taro3的vue跨端是否是用vue3的createRenderer功能实现的，如果不是，不使用的原因是什么呢？

### DISCUSSION THREAD

**Discussion by alamhubb** [NervJS/taro#15234] [Q & A]
### 这个特性解决了什么问题？
近期学习跨端知识了解到Remax是基于react-reconciler实现的跨端，taro3的react跨端是否也使用了react-reconciler 来实现的跨端

那么vue3的跨端是否是使用vue3类似于react-reconciler 的createRenderer实现的呢？

如果不是为什么没采用相关技术呢，想了解学习这方面的内容，相关方请教谢谢

### 这个 API 长什么样？
使用createRenderer实现vue3跨端

**Comment by Chen-jj** [comment]
@social-uni 可以的，使用 `createRenderer` 还可以更精确地处理 Vue 与小程序的差异 。只是刚适配 Vue3 的时候还没有这个功能，目前还没人力去适配这样子。

**Comment by alamhubb** [comment]
> @social-uni 可以的，使用 `createRenderer` 还可以更精确地处理 Vue 与小程序的差异 。只是刚适配 Vue3 的时候还没有这个功能，目前还没人力去适配这样子。

希望未来可以实现，谢谢，辛苦了
